src/full/Agda/Utils/ListT.hs

```diff
@@ -109,9 +109,13 @@ liftListT lift xs = runMListT $ caseMaybeM (lift $ runListT xs) (return nilListT
 instance Monad m => Semigroup (ListT m a) where
   l1 <> l2 = ListT $ foldListT cons (runListT l2) l1
     where cons a = runListT . consListT a . ListT
+
 instance Monad m => Monoid (ListT m a) where
+  {-# INLINE mempty #-}
```

Comment:
Did you mean to zap this?

Reply:
Argh.  Yes.  (I need to clean up my vim plug-ins; the summarising plug-in is not always your friend.)